_docs/resources/sketch-library.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Sketch Library
+layout: docs
+status: stable
+resource: true
+category: resources
+last-modified: 2018-04-05
+usage: >
+   The Helix team provides a rich set of preconfigured user interface (UI) components that you can use in Sketch design toolkit to design Rackspace Control Panels. These come in both high-fidelity (library) and low-fidelity variations (stickersheet).
+---
+
+## Sketch Library
+
+<section class="static-section" markdown="1">
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+There are two approaches to installing the Helix library:
+
+1. **You can install the Sketch Library from your Google Drive.** The primary
```

Comment:
`You can install the Sketch Library` is getting in the way of easy reading.

Change to something more direct:

**1. Use Google Drive sync on your computer**
**2. Download directly from GitHub**

Reply:
updated.